setup.cfg

```diff
@@ -41,8 +41,8 @@ install_requires =
     nltk==3.5
     numpy>=1.18
     openapi-schema-validator==0.1.4
-    openpyxl==3.0.3
-    pandas>=1.0
+    openpyxl==3.0.10
```

Comment:
I would do

    openpyl~=3.0
    pandas~=1.4

so we can still get bug fix releases of these packages without having to lift a finger.

Reply:
resolved by 7e9b7bf (tyty!)